### PR TITLE
302541 update git mirror refs

### PR DESCRIPTION
**Pull request by Nino-K**

### CODE REVIEW

File: deps.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
 {
     "confdis": {
         "repo": "http://git-mirrors.stackato.com/github.com/hpcloud/confdis.git",
-        "version": "280f9c36a",
+        "version": "430f574e6e585ed3ea90236ccd34009e33e06a5b",
         "type": "git-clone",
         "alias": "confdis"
     },
     "github.com/ActiveState/httpapi": {
         "repo": "github.com/hpcloud/httpapi",
```
Comment (mook-as) [inline deps.json]: I don't think `repo` without the scheme works with `git-clone`?

**Comment by philwhln** [comment]
We should migrate all the references s/ActiveState/hpcloud/g in deps.json

File: deps.json (hunk 1 above)
Comment (Nino-K) [inline deps.json]: Updated [here](https://github.com/hpcloud/stackato-go/commit/b4b976cd1f9ed7c257749268c623d742dd1edebe)

**Comment by mook-as** [comment]
That looks fine, but do you need to do the same for `log` and `run`?

**Comment by Nino-K** [comment]
I think so :-( , shall we just edit imports?

**Comment by mook-as** [comment]
That might be easiser :cry:

**Comment by Nino-K** [comment]
Closing this PR. Updated the references from ActiveState to hpcloud [here](https://github.com/hpcloud/stackato-go/pull/6)

**Pull request closed without merging**